telegram/_message.py

```diff
@@ -553,6 +554,11 @@ class Message(MaybeInaccessibleMessage):
 
             .. versionadded:: 21.1
 
+        chat_background_set  (:obj:`telegram.ChatBackground`, optional): Service message: chat
```

Comment:
we should add a new sub-filter for `filters.StatusUpdate` for this

Reply:
Done